tests/functional/NotifyQueueConsumerTest/Queue/ConsumerTest.php

```diff
@@ -238,7 +238,8 @@ private function createMessageWithFile(string $uuid, int $documentId, ?string $l
                     'Test Surname',
                     $letterType,
                     'OPG103',
-                    '74442574'
+                    '74442574',
+                    'HW'
```

Comment:
Is there a test for it not being HW?

Reply:
For the Consumer tests, these test the consumer running through the process from picking up the message and then ultimately updating the status. This function `createMessageWithFile` is just to emulate the message that is on the queue ready to be picked up. I've added the not being HW to the unit test for the handler as part of the `annualReportLetterData` provider.